Cython/Compiler/Symtab.py

```diff
@@ -7,6 +7,13 @@
 import re
 import copy
 import operator
+import sys
+
+is_openvms = (sys.platform == 'OpenVMS')
```

Comment:
We don't assume that the platform that Cython is run on is the same as the one that the C file is compiled on. Lots of projects distribute the .c files rather than the .pyx file.

I would think that Cython should just do what you tell it (even using a C keyword is only a Cython warning) and it should be up to the user to pick a name that works on their system. Might there be times when the user intentionally wants to use these reserved names?

Reply:
+1 to both of these statements.

Reply:
The issue is appeared while compiling NUMPY. There is a public member in Python which is named 'readonly'. In the Python port we renamed it to 'readonly$' in C files. But Python name remained 'readonly' of course. When cythonizing sources with this name we should use 'readonly$' in C files.

Reply:
We should still not be producing different C code depending on the platform on which Cython is run. It seems the right thing to do here if you have to rename things in one spot is rename them in others as well, rather than modify tools to do the matching rename for you, but I would be open to trying to explore the best solution here.